homeassistant/components/huawei_lte/__init__.py

```diff
@@ -623,6 +627,7 @@ async def async_update_options(self, config_entry: ConfigEntry) -> None:
 
     async def async_added_to_hass(self) -> None:
         """Connect to update signals."""
+        assert self.hass is not None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6309587)